experimental/uefi/runtime/src/remote_attestation.rs

```diff
@@ -31,6 +31,10 @@ use oak_remote_attestation_sessions::{SessionId, SessionState, SessionTracker};
 /// Number of sessions that will be kept in memory.
 const SESSIONS_CACHE_SIZE: usize = 10000;
 
+pub trait AttestationTrait {
```

Comment:
Fair, renamed the trait to `AttestationHandler` and the struct to `AttestationSessionHandler`. The trait is actually technically needed here, since it allows us to use a dynamic implementation of the trait at runtime. L37 main.rs.

Given that the WASM module is passed at runtime, we cannot know the precise signature of the struct inn advance. :)

Reply:
I only noticed this after my other comment. I don't think I understand, as it does not seem that the type of the struct that will be used depends on the Wasm module bytes, only on the closure that is used to handle requests (which seems to be fixed). I would have assumed adding an additional trait bound to account for the closure should be sufficient.

Reply:
I'm not sure trait bounds for closures are possible, afaik rust closures have unique, anonymous types. But it might be possible to achieve this by refactoring the attestation logic to instead of a closure, accept the implementation of a trait.